recipe/build-mpi.sh

```diff
@@ -78,13 +84,6 @@ fi
 make -j"${CPU_COUNT:-1}"
 make install
 
-# openmpi installs .mod files in the wrong prefix (/lib instead of /include)
```

Comment:
I'm wondering whether rather than remove this block, we should instead update it to link modules from $includedir into $libdir. My only concern is users setting things flags manually that may get a nasty surprise after this change. Remeber this is a patch release... WDYT?

Reply:
We can keep this and reverse the symlinks with a todo to remove them in 5.1 or 6. I do personally feel comfortable removing them here, since it takes a pretty wild combination of circumstances necessarily ignoring proper sources of compilation information for this to cause a problem, but we can save that for 5.1. Users would have to:

1. use the openmpi fortran modules, and
2. use compiler wrappers, and
3. not respect $CFLAGS, pkg-config, etc., and
4. not use _any_ includes other than openmpi fortran modules, which would mean $PREFIX/include is present already

My guess would be there is no code in the world that meets these criteria and builds with conda-forge openmpi, and the fix for any affected code is trivial.